Terminal.Gui/Views/FrameView.cs

```diff
@@ -1,68 +1,61 @@
-﻿using System;
-using System.Linq;
+﻿using System.Linq;
 using System.Text.Json.Serialization;
-using System.Text;
-using static Terminal.Gui.ConfigurationManager;
 
-namespace Terminal.Gui {
+namespace Terminal.Gui;
+/// <summary>
+/// The FrameView is a container frame that draws a frame around the contents. It is similar to
+/// a GroupBox in Windows.
+/// </summary>
+public class FrameView : View {
 	/// <summary>
-	/// The FrameView is a container frame that draws a frame around the contents. It is similar to
-	/// a GroupBox in Windows.
+	/// Initializes a new instance of the <see cref="Gui.FrameView"/> class using <see cref="LayoutStyle.Absolute"/> layout.
 	/// </summary>
-	public class FrameView : View {
-		/// <summary>
-		/// Initializes a new instance of the <see cref="Gui.FrameView"/> class using <see cref="LayoutStyle.Absolute"/> layout.
-		/// </summary>
-		/// <param name="frame">Frame.</param>
-		/// <param name="title">Title.</param>
-		/// <param name="views">Views.</param>
-		public FrameView (Rect frame, string title = null, View [] views = null) : base (frame)
-		{
-			SetInitialProperties (frame, title, views);
-		}
-
-		/// <summary>
-		/// Initializes a new instance of the <see cref="Gui.FrameView"/> class using <see cref="LayoutStyle.Computed"/> layout.
-		/// </summary>
-		/// <param name="title">Title.</param>
-		public FrameView (string title)
-		{
-			SetInitialProperties (Rect.Empty, title, null);
-		}
+	/// <param name="title">Title.</param>
+	/// <param name="views">Views.</param>
+	public FrameView (string title = null, View [] views = null)
```

Comment:
Why not nuke this one as well?

It's easy enough to do

```cs
var f = new FrameView() { Title = "Title" };
f.Add(views);
```

Reply:
I didn't encounter nothing of this. Maybe was already fixed before.